src/GovUk.Education.ExploreEducationStatistics.Data.Services/ResultSubjectMetaService.cs

```diff
@@ -93,49 +107,62 @@ public class ResultSubjectMetaService : AbstractSubjectMetaService, IResultSubje
                     _logger.LogTrace("Got Footnotes in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
                     stopwatch.Restart();
 
-                    var geographicLevels = locations.Keys.ToList();
-
-                    var geoJsonAvailable = HasBoundaryLevelDataForAnyGeographicLevel(geographicLevels);
-                    _logger.LogTrace("Got GeoJsonAvailable in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
-                    stopwatch.Restart();
-
-                    var boundaryLevelViewModels = GetBoundaryLevelViewModels(
-                        query.BoundaryLevel,
-                        locations.Keys);
-
                     var indicatorViewModels = GetIndicatorViewModels(query);
                     _logger.LogTrace("Got Indicators in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
                     stopwatch.Restart();
 
-                    var locationViewModels = GetLegacyLocationViewModels(
-                        locations,
-                        query.IncludeGeoJson ?? false,
-                        query.BoundaryLevel);
-                    _logger.LogTrace("Got Location view models in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
-                    stopwatch.Restart();
-
                     var timePeriodViewModels = GetTimePeriodViewModels(observations);
                     _logger.LogTrace("Got Time Periods in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
-                    stopwatch.Stop();
+                    stopwatch.Restart();
 
                     var publicationId = await _subjectRepository.GetPublicationIdForSubject(subject.Id);
                     var publicationTitle = (await _contentDbContext.Publications.FindAsync(publicationId)).Title;
 
                     var releaseFile = await _releaseDataFileRepository.GetBySubject(releaseId, subject.Id);
-                    var subjectName = releaseFile.Name;
-                    
-                    return new ResultSubjectMetaViewModel
+                    var subjectName = releaseFile.Name ?? "";
```

Comment:
Would this make more sense to use the null-forgiving operator, as we know that a ReleaseFile should certainly have a value by now?